docs/design/cincinnati.md

```diff
@@ -216,31 +216,31 @@ Determining valid graph traversals can be tricky to do by hand. The following [`
 
 #### Example ####
 
-The following will fetch the graph from a remote endpoint and determine the set of versions to which "1.0.0" can transition:
+The following will fetch the graph from the remote endpoint and determine the set of versions to which OpenShift version "4.2.0" can transition:
```

Comment:
I don't think we need the quotes, it can just be:

> ... to which OpenShift version 4.2.0 can transition.

Or maybe:

> ... the set of available updates currently recommended for OpenShift 4.2.0.

with an implicit "as claimed by whoever is running the Cincinnati service you're querying".

Reply:
@wking thanks. It is ready to be merged.

Reply:
Continued in https://github.com/openshift/cincinnati/pull/189#discussion_r375613320